python_practice/P002.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""Swap the values of two variables."""
+
+
+def swap_variables(val1, val2):
+    """Returns swapped variables."""
+    val = val1
```

Comment:
you can do it like this:
`val1, val2 = val2, val1`
the pattern is: `a, b = c, d`
the value of c will be assigned to a, the value of d will be assigned to b
so with `val1, val2 = val2, val1`,
the value of val2 will be assigned to val1 and the value of val1 will be assigned to val2